tests/ttnn/sweep_tests/debug_sweeps.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+
+import ttnn
+
+
+from tests.ttnn.sweep_tests.sweep import debug_sweeps
+
```

Comment:
Do we need documentation on how to add sweep tests and use the exclude?

Reply:
Yes, I will add it